eras/mary/impl/cardano-ledger-mary.cabal

```
@@ -60,7 +60,7 @@ library
         cardano-ledger-allegra >=1.1,
         cardano-ledger-binary >=1.0,
         cardano-ledger-core ^>=1.3,
-        cardano-ledger-shelley ^>=1.3,
+        cardano-ledger-shelley ^>=1.4,
```

Also needs a patch bump
```suggestion
        cardano-ledger-shelley  >=1.3 && <1.5,
```